src/views/GameListView.jsx

```diff
@@ -150,18 +270,63 @@ const GameListView = () => {
       <Content>
         <Filter>
           <h2 className="incentive">{t("Start a game now")}</h2>
+          <input
+            type="search"
+            id="game-search"
+            name="game-search"
+            aria-label={t("Search for a game")}
+            placeholder={t("Search for a game")}
+            value={filterCriteria.searchTerm}
+            onChange={onChangeSearchTerm}
+          />
           <StyledGameFilters>
-            <li>
-              <input
-                type="search"
-                id="game-search"
-                name="game-search"
-                aria-label={t("Search for a game")}
-                placeholder={t("Search for a game")}
-                value={searchTerm}
-                onChange={handleChange}
+            <li className="player-filter">
+              <span className="filter-title">{t("Number of players")}</span>
+              <SliderRange
+                defaultValue={[1, 9]}
+                value={filterCriteria.nbOfPlayers}
+                step={1}
+                onChange={onChangeNbOfPlayersSlider}
               />
             </li>
+            <li className="duration-filter">
+              <span className="filter-title">{t("Duration (mins)")}</span>
+              <SliderRange
+                defaultValue={[15, 90]}
+                value={filterCriteria.durations}
+                step={15}
+                onChange={onChangeDurationSlider}
+              />
+            </li>
+            <li className="small-filter language-filter">
+              <span className="filter-title">{t("Language")}</span>
+              <div>
+                <ul>
+                  <li>
+                    <input
+                      type="checkbox"
+                      id="french-filter"
+                      name="french-filter"
+                      defaultChecked={filterCriteria.languages.includes("fr")}
+                      onChange={onChangelanguageFilter}
+                      data-lang="fr"
+                    ></input>
+                    <label htmlFor="french-filter">{t("French")}</label>
+                  </li>
+                  <li>
+                    <input
+                      type="checkbox"
+                      id="english-filter"
+                      name="english-filter"
+                      defaultChecked={filterCriteria.languages.includes("en")}
+                      onChange={onChangelanguageFilter}
+                      data-lang="en"
+                    ></input>
```

Comment:
```suggestion
                    />
```
for other inputs also.